tests/molecular/key_makers/conftest.py

```diff
@@ -18,6 +18,12 @@
             key_name='InChIKey',
             key='PIICEJLVQHRZGT-UHFFFAOYSA-N',
         ),
+        CaseData(
+            key_maker=stk.Smiles(),
+            molecule=stk.BuildingBlock('NCCN'),
+            key_name='SMILES',
+            key='NCCN',
+        ),
```

Comment:
Maybe a test case to check that the smiles is indeed canonical, and another test case to check that the smiles is isomeric.
FOr the canonical test case if you make a building block like 
`BuildingBlock('C(N)CN')` you should get `'NCCN'` as the smiles also.